### PR TITLE
102: User logout functionality

### LINKED ISSUE #102: Implement user logout functionality
**Description:**

When a user is logged in and clicks the logout button, they should be logged out of the app.

**Acceptance Criteria:**

- [x] Clear state and navigate to the login page on logout button click

### PR DESCRIPTION
**Pull request by ZackHolmberg**
closes #102

### CODE REVIEW

**Review by abhisachdev17 — APPROVED** [review]
LGTM

File: src/frontend/src/components/NavBar.vue [hunk 1]
```diff
@@ -2,24 +2,28 @@
...
       />
     </router-link>
-    <img
-      id="nav-logout"
-      class="nav-logout"
-      alt="Logout icon"
-      src="../assets/logoutIcon.png"
-    />
+    <div @click="logout">
+      <router-link to="/account">
```
Comment (madisonfines) [inline src/frontend/src/components/NavBar.vue]: Don't think you need this to=/account here!

**Review by madisonfines — APPROVED** [review]
approve

File: src/frontend/src/components/NavBar.vue (hunk 1 above)
Comment (madisonfines) [reply src/frontend/src/components/NavBar.vue]: I think you can just remove router-link from here completely actually

Comment (ZackHolmberg) [reply src/frontend/src/components/NavBar.vue]: Good catch!

**Pull request merged**